VMware.CloudFoundation.CertificateManagement.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule = '.\VMware.CloudFoundation.CertificateManagement.psm1'
 
     # Version number of this module.
-    ModuleVersion = '1.1.0.1002'
+    ModuleVersion = '1.2.0.1001'
```

Comment:
```suggestion
    ModuleVersion = '1.2.0.1000'
```

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/45/commits/48ca113f5f0d3edc9163c227b3b3871f636f2e81.